src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_validators.py

```diff
@@ -4,12 +4,12 @@
 # --------------------------------------------------------------------------------------------
 
 from knack.util import CLIError
-from ._factory import get_acr_service_client
+from ._client_factory import get_acr_service_client
 
 
 def validate_registry_name(namespace):
     if namespace.registry_name:
-        client = get_acr_service_client().registries
+        client = get_acr_service_client(namespace.cmd.cli_ctx).registries
```

Comment:
Avoid doing this.  Instead, put cmd in the validator signature and reference that.

Reply:
Didn't know cmd was added automatically.
Thanks.